src/ekomark/benchmark/runner.py

```diff
@@ -14,6 +14,7 @@
 
 import eko
 from eko import basis_rotation as br
+from ekobox import apply
```

Comment:
Here we have a problem: we're using `ekobox` in `ekomark`, but the dependency is supposed to go the other way round.

Consider that the database and `navigator` are defined inside `ekomark`, but we want to expose them in the UI.
On the other hand, what it's done here is intuitive, so we really need a more involved solution...

(I wonder if it's worth to split the PDF dumping in a further separate package...)

Reply:
or we just move the apply stuff inside `ekomark`? if this is the first place we need it ... (and it is a benchmark - with `lhapdf`)

Reply:
Do we want to move apply in ekomark then?

Reply:
Even fine, let's do this

Reply:
Ok, should be fine now.